CI.hs

```diff
@@ -267,10 +308,10 @@ buildDists ghcFlavor resolver = do
         putStrLn $ "# Completed in " ++ showDuration t ++ ": " ++ x ++ "\n"
         hFlush stdout
 
-      mkTarball :: String -> String -> IO ()
```

Comment:
Are you sure that the resolver doesn’t matter here? Seems safer to add it if it doesn’t cause problems.

Reply:
The change here is  that `mkTarball` now calls the function `stack` (rather than `cmd "stack"` directly) so indeed,  the resolver is taken into account.

Reply:
Oh makes sense. Thanks for the explanation!